gridt/src/app/home/home.page.scss

```diff
@@ -10,11 +10,8 @@
 .movement-card{
   min-width: 300px;
 }
-
-// The width of cards on the home page should match the width of the screen for small enough screens. Issue #60
-@media screen and (max-width: 576px) {
-  .movement-card{
-    width: 100%;
-    margin: 0;
-  }
+.movement-card
+{
+     margin: 10px;
```

Comment:
The indentation is inconsistent in this document. The standard in this project is an identation of 2.